example/software/application.json

```diff
@@ -1,6 +1,15 @@
 {
+    "_id": "K3McGkqXh28HJ775m",
     "name": "espresso",
     "shortName": "qe",
     "summary": "Quantum Espresso",
-    "version": "5.1.1"
+    "version": "5.1.1",
+    "isDefault": true,
+    "owner": {
+        "...": "include(../system/owner.json)"
+    },
+    "schemaVersion": "0.2.0",
+    "creator": {
+        "...": "include(../system/creator.json)"
+    }
```

Comment:
Per our conversation about pure schemas and schemas used in the webapp (InMemoryEntity vs Entity) - maybe it's better to combine the pure schemas (for Application, Flavor, Template, Material, etc) with Entity-based keys (`owner, creator, isDefault`) in a single place to avoid sprinkling the additional values everywhere?